apps/cli/README.md

```diff
@@ -38,6 +38,8 @@ bun create arweave-app@latest
 
 During the interactive setup, you'll be prompted for your project's name and other configuration options. Provide your choices to create a new Arweave application.
 
+> If using a secure shell, ensure your ssh-agent is running as expected for successfull installation of dependencies.
```

Comment:
Also maybe we place it somewhere as this issue can be present in interactive or non-interactive mode. Also, maybe mentioning its an issue in windows can be helpful.